appveyor.yml

```diff
@@ -44,19 +49,30 @@ build:
 
 after_build:
   - cmd: rmdir /s /q C:\projects\native-algorithms\src\packages
+  - cmd: 7z a cached_artifacts.zip src\Win32 src\x64 & exit 0
```

Comment:
Is this `& exit 0` really necessary here? According to [this source](http://7zip.bugaco.com/7zip/MANUAL/exit_codes.htm), 7z should work fine on its own.

Reply:
During the first job, we won't have all artifacts. It returns `1` then, instead of `0`.